nuxeo-ai-model/src/main/java/org/nuxeo/ai/cloud/NuxeoCloudClient.java

```diff
@@ -98,10 +98,12 @@ public void start(ComponentContext context) {
 
     protected void initClient() {
         List<CloudConfigDescriptor> configs = getDescriptors(XP_CONFIG);
-        if (configs.size() == 1) {
-            configureClient(configs.get(0));
-        } else {
-            throw new IllegalArgumentException("Nuxeo cloud client requires 1 single configuration.");
+        if (!configs.isEmpty()) {
+            if (configs.size() == 1) {
+                configureClient(configs.get(0));
+            } else {
+                throw new IllegalArgumentException("Nuxeo cloud client requires 1 single configuration.");
+            }
```

Comment:
you can fully inline it

```java
if (configs.isEmpty()) {
    // NOP
} else if (configs.size() == 1) {
    configureClient(configs.get(0));
} else {
    throw new IllegalArgumentException("Nuxeo cloud client requires 1 single configuration.");
}

Reply:
it's more like it was initially done by Gethin. No real preferences here on my side